.pipelines/cni/cilium/cilium-scale-test.yaml

```diff
@@ -2,6 +2,63 @@ pr: none
+    jobs:
+      - job: create_rg_cluster
+        pool:
+          name: "$(BUILD_POOL_NAME_DEFAULT)"
+        steps:
+          - task: AzureCLI@2
+            inputs:
+              azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              addSpnToEnvironment: true
+              inlineScript: |
+                set -ex
+                az extension add --name aks-preview
+                if [ ${CREATE_RG} ]; then
+                  az group create --name ${RESOURCE_GROUP} --location ${LOCATION}
+                fi
+                subscriptionId="$(az account list --query "[?isDefault].id" --output tsv)"
+                make -C ./hack/aks AZCLI=az ${CLUSTER_TYPE} CLUSTER=${CLUSTER} GROUP=${RESOURCE_GROUP} REGION=${LOCATION} NODE_COUNT=5 VM_SIZE=${VMSIZE} SUB=${subscriptionId}
+                ls -lah
+                pwd
+                kubectl cluster-info
+                kubectl get po -owide -A
+
+                echo "install Cilium ${CILIUM_VERSION_TAG}"
+                export DIR=${CILIUM_VERSION_TAG%.*}
+                echo "installing files from ${DIR}"
+
+                echo "deploy Cilium ConfigMap"
+                if [ ${USE_DUALSTACK} ]; then
+                  echo "Use dualstack configmap for Cilium"
+                  kubectl apply -f test/integration/manifests/cilium/v${DIR}/cilium-config/cilium-config-dualstack.yaml
+                else
+                  kubectl apply -f test/integration/manifests/cilium/v${DIR}/cilium-config/cilium-config.yaml
+                fi
+
+                # Passes Cilium image to daemonset and deployment
+                kubectl apply -f test/integration/manifests/cilium/v${DIR}/cilium-agent/files
+                kubectl apply -f test/integration/manifests/cilium/v${DIR}/cilium-operator/files
+
+                if [ ${USE_DUALSTACK} ]; then
+                  echo "Use dualstack daemonset for Cilium"
+                  export IPV6_HP_BPF_VERSION=$(make ipv6-hp-bpf-version)
+                  envsubst '${CILIUM_VERSION_TAG},${CILIUM_IMAGE_REGISTRY},${IPV6_HP_BPF_VERSION}' < test/integration/manifests/cilium/v${DIR}/cilium-agent/templates/daemonset-dualstack.yaml | kubectl apply -f -
+                else
+                  envsubst '${CILIUM_VERSION_TAG},${CILIUM_IMAGE_REGISTRY}' < test/integration/manifests/cilium/v${DIR}/cilium-agent/templates/daemonset.yaml | kubectl apply -f -
+                fi
+
+                envsubst '${CILIUM_VERSION_TAG},${CILIUM_IMAGE_REGISTRY}' < test/integration/manifests/cilium/v${DIR}/cilium-operator/templates/deployment.yaml | kubectl apply -f -
+                kubectl get po -owide -A
+
+                echo "Deploy Azure-CNS"
+                sudo -E env "PATH=$PATH" make test-integration AZURE_IPAM_VERSION=v$(AZURE_IPAM_VERSION) CNS_VERSION=v$(CNS_VERSION) INSTALL_CNS=true INSTALL_OVERLAY=true CNS_IMAGE_REPO=$(CNS_IMAGE_REPO)
```

Comment:
maybe instead of running integration tests we can apply the cns manifests and inject the env variables similar to how we do it for cilium? 

`test-integration` might cause flakes early in the pipeline